lib/webauthn/public_key_credential_with_assertion.rb

```diff
@@ -31,5 +32,14 @@ def user_handle
     def raw_user_handle
       response.user_handle
     end
+
+    private
+
+    def appid
+      appid_domain = WebAuthn.configuration.appid || WebAuthn.configuration.rp_id ||
+                     WebAuthn.configuration.origin || raise("Unspecified expected origin")
```

Comment:
I think we should find a way of unifying the identification of where the `rp_id` comes from in one method available for all, cause we are doing something similar in [here](https://github.com/cedarcode/webauthn-ruby/blob/e926b6a1f9a58419c96911be460215eefe60d738/lib/webauthn/authenticator_response.rb#L36).